manifests/init.pp

```diff
@@ -102,8 +105,9 @@
   Stdlib::Port $api_port = 24817,
   Stdlib::Host $content_host = '127.0.0.1',
   Stdlib::Port $content_port = 24816,
-  Stdlib::Absolutepath $webserver_static_dir = '/var/lib/pulp/docroot',
-  Stdlib::Absolutepath $pulp_static_root = '/var/lib/pulp/assets',
+  Stdlib::Absolutepath $apache_docroot = $user_home,
+  Stdlib::Absolutepath $pulpcore_media_root = $user_home,
+  Stdlib::Absolutepath $pulpcore_static_root = "${apache_docroot}/assets",
```

Comment:
I just found out that Apache refuses to serve `pulpcore_var_lib_t` so this needs another SELinux policy change. We should agree on a layout and properly label that.